src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -135,43 +247,167 @@ public void run() {
 
     @Override
     public Channel getChannel(ThingUID thingUID, ZigBeeEndpoint endpoint) {
-        if (endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID) == null
+        clusterColorControl = (ZclColorControlCluster) endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID);
+
+        if (clusterColorControl == null || endpoint.getInputCluster(ZclOnOffCluster.CLUSTER_ID) == null
                 || endpoint.getInputCluster(ZclLevelControlCluster.CLUSTER_ID) == null) {
             return null;
         }
+
+        try {
+            if(!clusterColorControl.discoverAttributes(false).get()) {
+                logger.warn("{}: Cannot determine whether device endpoint supports RGB color. Assuming it does by now (checking again later)", endpoint.getEndpointAddress());
+            }
+            else if(clusterColorControl.getSupportedAttributes().contains(ZclColorControlCluster.ATTR_CURRENTHUE)) {
+                logger.debug("{}: Device endpoint supports HUE/SATURATION color", endpoint.getEndpointAddress());
+            }
+            else if(clusterColorControl.getSupportedAttributes().contains(ZclColorControlCluster.ATTR_CURRENTX)) {
+                logger.debug("{}: Device endpoint supports XY color", endpoint.getEndpointAddress());
+            }
+            else {
+                logger.debug("{}: Device endpoint does not support RGB color (probably only color temperature)", endpoint.getEndpointAddress());
```

Comment:
Does this really need this log entry? I don't think we log a "channel not supported" type message for all other converters.

Reply:
Useful for me to debug while coding, but I can remove it for the PR